modules/risk-view-details.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operating/evaluate-security-risks.adoc
+:_module-type: CONCEPT
+[id="risk-view-details_{context}"]
+= Risk view
+
+[role="_abstract"]
+The *Risk* view lists all deployments from all clusters, sorted by a multi-factor risk metric based on policy violations, image contents, deployment configuration, and other similar factors.
+Deployments at the top of the list present the most risk.
+
+*Risk* view shows list of deployments with following attributes for each row:
```

Comment:
_The_ **Risk** view shows _a_ list of deployments with _the_ following attributes for each row: